nixos-x86_64.json

```diff
@@ -52,6 +52,33 @@
           "1024"
         ]
       ]
+    },
+    {
+      "boot_wait": "40s",
+      "boot_command": [
+        "echo http://{{.HTTPIP}}:{{.HTTPPort}} > .packer_http<enter>",
+        "mkdir -m 0700 .ssh<enter>",
+        "curl $(cat .packer_http)/install_rsa.pub > .ssh/authorized_keys<enter><wait>",
+        "nixos-rebuild switch<enter><wait>",
+        "systemctl start sshd<enter>"
+      ],
+      "http_directory": "scripts",
+      "switch_name": "Devbox",
+      "iso_checksum_type": "sha256",
+      "shutdown_command": "shutdown -h now",
+      "ssh_private_key_file": "./scripts/install_rsa",
+      "ssh_port": 22,
+      "ssh_timeout": "600s",
+      "ssh_username": "root",
+      "type": "hyperv-iso",
+      "iso_url": "https://d3g5gsiof5omrk.cloudfront.net/nixos/18.09/nixos-18.09.1799.b9fa31cea0e/nixos-minimal-18.09.1799.b9fa31cea0e-x86_64-linux.iso",
+      "iso_checksum": "cc7c399c5fe4672383fe54cb1d648854a0d6732765fe1a61bb38b3fe3b7c6d2f",
+      "disk_size": 62000,
+      "cpu": 3,
+      "ram_size": 2048,
+      "enable_mac_spoofing": true,
+      "guest_additions_mode": "disable",
+      "enable_virtualization_extensions": false
```

Comment:
The json code is generated by the gen_template.rb script. The new target would be declared here: https://github.com/nix-community/nixbox/blob/dcff88c4dffbcb501856e50bf55946365485c41e/gen_template.rb#L74

Reply:
To re-generate the json run `nix-shell --run make update`